pkgs/development/tools/misc/texinfo/6.0.nix

```diff
@@ -9,8 +9,11 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ perl xz ]
+    ++ stdenv.lib.optional stdenv.isSunOS libiconv
     ++ stdenv.lib.optional interactive ncurses;
 
+  configureFlags = "AWK=${gawk}/bin/awk";
```

Comment:
I would assume whole `/usr/bin` shouldn't be on `$PATH` by default, even if it's accessible. `./configure` typically prefers searching in `$PATH` first, I expect.